scheduler/api/server.go

```diff
@@ -31,50 +32,106 @@ func MakeServer(handler scoot.CloudScoot,
 	h.stat.Counter(stats.SchedServerJobKillCounter).Inc(1)
-	return schedthrift.KillJob(jobId, h.scheduler, h.sagaCoord)
+	return schedthrift.KillJob(jobID, h.scheduler, h.sagaCoord)
 }
 
-// Implements OfflineWorker Cloud Scoot API
+// OfflineWorker Implements OfflineWorker Cloud Scoot API
 func (h *Handler) OfflineWorker(req *scoot.OfflineWorkerReq) error {
 	return schedthrift.OfflineWorker(req, h.scheduler)
 }
 
-// Implements ReinstateWorker Cloud Scoot API
+// ReinstateWorker Implements ReinstateWorker Cloud Scoot API
 func (h *Handler) ReinstateWorker(req *scoot.ReinstateWorkerReq) error {
 	return schedthrift.ReinstateWorker(req, h.scheduler)
 }
 
-// Implements GetSchedulerStatus Cloud Scoot API
+// GetSchedulerStatus Implements GetSchedulerStatus Cloud Scoot API
 func (h *Handler) GetSchedulerStatus() (*scoot.SchedulerStatus, error) {
 	return schedthrift.GetSchedulerStatus(h.scheduler)
 }
 
-// Implements SetSchedulerStatus Cloud Scoot API
+// SetSchedulerStatus  Implements SetSchedulerStatus Cloud Scoot API
 func (h *Handler) SetSchedulerStatus(maxNumTasks int32) error {
 	return schedthrift.SetSchedulerStatus(h.scheduler, maxNumTasks)
 }
+
+// GetClassLoadPcts Implements GetClassLoadPcts Cloud Scoot API
+func (h *Handler) GetClassLoadPcts() (map[string]int32, error) {
+	clp, err := schedthrift.GetClassLoadPcts(h.scheduler)
+	log.Infof("GetClassLoadPcts returning: %v, err:%s", clp, err)
+	return clp, err
+}
+
+// SetClassLoadPcts Implements SetClassLoadPcts Cloud Scoot API
+func (h *Handler) SetClassLoadPcts(classLoadPcts map[string]int32) error {
+	log.Infof("SetClassLoadPcts to %v", classLoadPcts)
+	return schedthrift.SetClassLoadPcts(h.scheduler, classLoadPcts)
+}
+
+// GetRequestorToClassMap Implements GetRequestorToClassMap Cloud Scoot API
+func (h *Handler) GetRequestorToClassMap() (map[string]string, error) {
+	rm, err := schedthrift.GetRequestorToClassMap(h.scheduler)
+	log.Infof("GetClassLoadPcts returning: %v, err:%s", rm, err)
+	return rm, err
+}
+
+// SetRequestorToClassMap Implements SetRequestorToClassMap Cloud Scoot API
+func (h *Handler) SetRequestorToClassMap(requestToClassMap map[string]string) error {
+	log.Infof("SetRequestorToClassMap to %v", requestToClassMap)
+	return schedthrift.SetRequestorToClassMap(h.scheduler, requestToClassMap)
+}
+
+// GetRebalanceMinDuration get the duration(minutes) that the scheduler needs to be exceeding
+// the rebalance threshold before rebalancing.  <= 0 implies no rebalancing
+func (h *Handler) GetRebalanceMinDuration() (int32, error) {
+	d, err := schedthrift.GetRebalanceMinDuration(h.scheduler)
+	log.Infof("GetRebalanceMinDuration returning: %d, err:%s", d, err)
+	return d, err
+}
+
+// SetRebalanceMinDuration set the duration(minutes) that the scheduler needs to be exceeding
+// the rebalance threshold before rebalancing.  <= 0 implies no rebalancing
+func (h *Handler) SetRebalanceMinDuration(durationMin int32) error {
```

Comment:
Could durationMin be a time.Duration? We could then convert that to an int32 if that's require by thrift.